.eslintrc.yaml

```diff
@@ -64,10 +64,12 @@ rules:
     lines-between-class-members: off # be more lenient on member declarations
     max-classes-per-file: off # helper classes are common
     max-len: [warn, 150] # more lenient on max length per line
-    new-cap: off # some class still doesn't follow this yet
-    no-underscore-dangle: off # allow underscores
+    new-cap: error
```

Comment:
If there are exceptions, just `/* eslint-disable-next-line new-cap */`

Reply:
every `off` specified here is to shut down the default enabled behavior. so if you need some rule to be kept around, just remove the line.